tests/drivers/disk/testcase.yaml

```diff
@@ -1,5 +1,8 @@
 tests:
   drivers.disk.usdhc:
+    harness: ztest
+    harness_config:
+      fixture: disk_sdcard
```

Comment:
Let's use `fixture: fixture_sdhc` to match `samples/subsys/fs/fat_fs/sample.yaml`

Reply:
Fixed